charts/pulsar/templates/presto/presto-coordinator-statefulset.yaml

```diff
@@ -202,6 +208,16 @@ spec:
               name: ca
               readOnly: true
             {{- end}}
+            {{- if and .Values.tls.enabled (or .Values.tls.broker.enabled (or .Values.tls.zookeeper.enabled .Values.tls.bookie.enabled)) }}
```

Comment:
Does presto talk to bk and zk directly?

Reply:
yes, presto get bk address from zk and actually directly read data from bookkeeper